koala.gemspec

```diff
@@ -23,7 +23,8 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.1'
 
-  gem.add_runtime_dependency("faraday", "< 2")
+  gem.add_runtime_dependency("faraday")
```

Comment:
Contrary to what I first though we can have faraday 1 and faraday 2 compatibility at the same time